Src/Support/Google.Apis.Auth.Tests/JsonWebSignatureTests.cs

```diff
@@ -77,12 +77,21 @@ public async Task WrongAlgorithm()
             Assert.NotNull(await JsonWebSignature.VerifySignedTokenAsync(
                 FakeCertificateCache.JwtGoogleSigned, BuildOptions()));
 
+#if NETCOREAPP2_0 || net452 || net46
         [Fact]
-        public void ValidES256Signature()
+        public async Task ValidES256Signature() =>
+            Assert.NotNull(await JsonWebSignature.VerifySignedTokenAsync(
+                FakeCertificateCache.Es256ForIap, BuildOptions(new MockClock { UtcNow = FakeCertificateCache.ValidEs256ForIap })));
+#else
```

Comment:
Possibly make this `#elif NETCOREAPP1.0 || NETCOREAPP1.1` and then add a `#else` block with `#error`? I think it's worth making sure we always deliberately choose one of these tests or the other.